CHANGELOG/CHANGELOG-0.5.md

```diff
@@ -1,3 +1,17 @@
+## v0.5.3
+
+Changes since `v0.5.2`:
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix Kueue crashing at the log level 6 when re-admitting workloads (#1645, @mimowo)
+- Kueue replicas are advertised as Ready only once the webhooks are functional.
+  
+  This allows users to wait with the first requests until the Kueue deployment is available, so that the early requests don't fail. (#1682 #1713, @mimowo @trasc)
+- Remove deleted pending workloads from the cache (#1687, @astefanutti)
```

Comment:
Also include https://github.com/kubernetes-sigs/kueue/pull/1617. This was also cherry-picked